lib/ansible/modules/cloud/softlayer/sl_vm.py

```diff
@@ -149,8 +149,7 @@
   gather_facts: False
   tasks:
   - name: Build instance request
-    local_action:
```

Comment:
That's remote, running against localhost (which makes the action to be performed... on localhost)